engines/glk/scott/scott.cpp

```diff
@@ -140,7 +140,8 @@ void Scott::runGame() {
 	// Check for savegame
 	_saveSlot = ConfMan.hasKey("save_slot") ? ConfMan.getInt("save_slot") : -1;
 
-	_G(_initialState) = saveCurrentState();
+	if(!shouldQuit())
```

Comment:
Please fix the formatting here: `if (!shouldQuit())`. 

Our formatting conventions can be found here: https://wiki.scummvm.org/index.php?title=Code_Formatting_Conventions

Reply:
Done!